tests/e2e/slashing.go

```diff
@@ -39,15 +39,22 @@ func (s *CCVTestSuite) TestRelayAndApplySlashPacket() {
 
 	for _, tc := range testCases {
 
+		// Reset test state
 		s.SetupTest()
-		s.SetupCCVChannel()
-		s.SetupTransferChannel()
```

Comment:
This previous call to `s.SetupTransferChannel()` was a no-op and is not included in the new diff